src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -136,9 +304,36 @@ public int countHeaders() {
     return headers.size();
   }
 
-  public String getBackgroundColor() {
-    wait.forElementVisible(component.get(0));
-    return component.get(0).getCssValue("background-color");
+  public InfoboxBuilderPage changeHeaderCollapsibilityState(int index) {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
```

Comment:
this line: ```JavascriptExecutor js = (JavascriptExecutor) driver;```
is not needed, `driver` object is already inheriting `JavascriptExecutor` class. Just use:
```driver.executeScript(script).toString();``` :)